### ISSUE TITLE
C4.4: Installation per Manager schlägt fehl

### ISSUE THREAD

**Issue by zonky2** [exploreimpact/contao-i18nl10n#122]
C4.4: Installation per Manager schlägt fehl

```
The background task has stopped unexpectedly. Please check the console protocol.
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - exploreimpact/contao-i18nl10n 1.5.0 requires contao/core >=3.2-dev,<4-dev -> satisfiable by contao/core[3.2.0, 3.2.1, 3.2.10, 3.2.11, 3.2.12, 3.2.13, 3.2.14, 3.2.15, 3.2.16, 3.2.17, 3.2.18, 3.2.19, 3.2.2, 3.2.20, 3.2.21, 3.2.3, 3.2.4, 3.2.5, 3.2.6, 3.2.7, 3.2.8, 3.2.9, 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.3.4, 3.3.5, 3.3.6, 3.3.7, 3.4.0, 3.4.1, 3.4.2, 3.4.3, 3.4.4, 3.4.5, 3.5.0, 3.5.1, 3.5.10, 3.5.11, 3.5.12, 3.5.13, 3.5.14, 3.5.15, 3.5.16, 3.5.17, 3.5.18, 3.5.19, 3.5.2, 3.5.20, 3.5.21, 3.5.22, 3.5.23, 3.5.24, 3.5.25, 3.5.26, 3.5.27, 3.5.28, 3.5.29, 3.5.3, 3.5.30, 3.5.31, 3.5.32, 3.5.33, 3.5.34, 3.5.35, 3.5.4, 3.5.5, 3.5.6, 3.5.7, 3.5.8, 3.5.9].
    - exploreimpact/contao-i18nl10n 1.5.1 requires contao/core >=3.2-dev,<4-dev -> satisfiable by contao/core[3.2.0, 3.2.1, 3.2.10, 3.2.11, 3.2.12, 3.2.13, 3.2.14, 3.2.15, 3.2.16, 3.2.17, 3.2.18, 3.2.19, 3.2.2, 3.2.20, 3.2.21, 3.2.3, 3.2.4, 3.2.5, 3.2.6, 3.2.7, 3.2.8, 3.2.9, 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.3.4, 3.3.5, 3.3.6, 3.3.7, 3.4.0, 3.4.1, 3.4.2, 3.4.3, 3.4.4, 3.4.5, 3.5.0, 3.5.1, 3.5.10, 3.5.11, 3.5.12, 3.5.13, 3.5.14, 3.5.15, 3.5.16, 3.5.17, 3.5.18, 3.5.19, 3.5.2, 3.5.20, 3.5.21, 3.5.22, 3.5.23, 3.5.24, 3.5.25, 3.5.26, 3.5.27, 3.5.28, 3.5.29, 3.5.3, 3.5.30, 3.5.31, 3.5.32, 3.5.33, 3.5.34, 3.5.35, 3.5.4, 3.5.5, 3.5.6, 3.5.7, 3.5.8, 3.5.9].
    - exploreimpact/contao-i18nl10n 1.5.2 requires contao/core >=3.2-dev,<4-dev -> satisfiable by contao/core[3.2.0, 3.2.1, 3.2.10, 3.2.11, 3.2.12, 3.2.13, 3.2.14, 3.2.15, 3.2.16, 3.2.17, 3.2.18, 3.2.19, 3.2.2, 3.2.20, 3.2.21, 3.2.3, 3.2.4, 3.2.5, 3.2.6, 3.2.7, 3.2.8, 3.2.9, 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.3.4, 3.3.5, 3.3.6, 3.3.7, 3.4.0, 3.4.1, 3.4.2, 3.4.3, 3.4.4, 3.4.5, 3.5.0, 3.5.1, 3.5.10, 3.5.11, 3.5.12, 3.5.13, 3.5.14, 3.5.15, 3.5.16, 3.5.17, 3.5.18, 3.5.19, 3.5.2, 3.5.20, 3.5.21, 3.5.22, 3.5.23, 3.5.24, 3.5.25, 3.5.26, 3.5.27, 3.5.28, 3.5.29, 3.5.3, 3.5.30, 3.5.31, 3.5.32, 3.5.33, 3.5.34, 3.5.35, 3.5.4, 3.5.5, 3.5.6, 3.5.7, 3.5.8, 3.5.9].
    - exploreimpact/contao-i18nl10n 1.5.3 requires contao/core >=3.2-dev,<4-dev -> satisfiable by contao/core[3.2.0, 3.2.1, 3.2.10, 3.2.11, 3.2.12, 3.2.13, 3.2.14, 3.2.15, 3.2.16, 3.2.17, 3.2.18, 3.2.19, 3.2.2, 3.2.20, 3.2.21, 3.2.3, 3.2.4, 3.2.5, 3.2.6, 3.2.7, 3.2.8, 3.2.9, 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.3.4, 3.3.5, 3.3.6, 3.3.7, 3.4.0, 3.4.1, 3.4.2, 3.4.3, 3.4.4, 3.4.5, 3.5.0, 3.5.1, 3.5.10, 3.5.11, 3.5.12, 3.5.13, 3.5.14, 3.5.15, 3.5.16, 3.5.17, 3.5.18, 3.5.19, 3.5.2, 3.5.20, 3.5.21, 3.5.22, 3.5.23, 3.5.24, 3.5.25, 3.5.26, 3.5.27, 3.5.28, 3.5.29, 3.5.3, 3.5.30, 3.5.31, 3.5.32, 3.5.33, 3.5.34, 3.5.35, 3.5.4, 3.5.5, 3.5.6, 3.5.7, 3.5.8, 3.5.9].
    - exploreimpact/contao-i18nl10n 1.5.4 requires contao/core >=3.2-dev,<4-dev -> satisfiable by contao/core[3.2.0, 3.2.1, 3.2.10, 3.2.11, 3.2.12, 3.2.13, 3.2.14, 3.2.15, 3.2.16, 3.2.17, 3.2.18, 3.2.19, 3.2.2, 3.2.20, 3.2.21, 3.2.3, 3.2.4, 3.2.5, 3.2.6, 3.2.7, 3.2.8, 3.2.9, 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.3.4, 3.3.5, 3.3.6, 3.3.7, 3.4.0, 3.4.1, 3.4.2, 3.4.3, 3.4.4, 3.4.5, 3.5.0, 3.5.1, 3.5.10, 3.5.11, 3.5.12, 3.5.13, 3.5.14, 3.5.15, 3.5.16, 3.5.17, 3.5.18, 3.5.19, 3.5.2, 3.5.20, 3.5.21, 3.5.22, 3.5.23, 3.5.24, 3.5.25, 3.5.26, 3.5.27, 3.5.28, 3.5.29, 3.5.3, 3.5.30, 3.5.31, 3.5.32, 3.5.33, 3.5.34, 3.5.35, 3.5.4, 3.5.5, 3.5.6, 3.5.7, 3.5.8, 3.5.9].
    - exploreimpact/contao-i18nl10n 1.5.5 requires contao/core >=3.2-dev,<4-dev -> satisfiable by contao/core[3.2.0, 3.2.1, 3.2.10, 3.2.11, 3.2.12, 3.2.13, 3.2.14, 3.2.15, 3.2.16, 3.2.17, 3.2.18, 3.2.19, 3.2.2, 3.2.20, 3.2.21, 3.2.3, 3.2.4, 3.2.5, 3.2.6, 3.2.7, 3.2.8, 3.2.9, 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.3.4, 3.3.5, 3.3.6, 3.3.7, 3.4.0, 3.4.1, 3.4.2, 3.4.3, 3.4.4, 3.4.5, 3.5.0, 3.5.1, 3.5.10, 3.5.11, 3.5.12, 3.5.13, 3.5.14, 3.5.15, 3.5.16, 3.5.17, 3.5.18, 3.5.19, 3.5.2, 3.5.20, 3.5.21, 3.5.22, 3.5.23, 3.5.24, 3.5.25, 3.5.26, 3.5.27, 3.5.28, 3.5.29, 3.5.3, 3.5.30, 3.5.31, 3.5.32, 3.5.33, 3.5.34, 3.5.35, 3.5.4, 3.5.5, 3.5.6, 3.5.7, 3.5.8, 3.5.9].
    - contao/core 3.2.0 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.1 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.10 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.11 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.12 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.13 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.14 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.15 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.16 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.17 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.18 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.19 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.2 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.20 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.21 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.3 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.4 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.5 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.6 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.7 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.8 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.2.9 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.3.0 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.3.1 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.3.2 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.3.3 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.3.4 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.3.5 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.3.6 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.3.7 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.4.0 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.4.1 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.4.2 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.4.3 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.4.4 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.4.5 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.0 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.1 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.10 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.11 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.12 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.13 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.14 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.15 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.16 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.17 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.18 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.19 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.2 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.20 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.21 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.22 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.23 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.24 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.25 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.26 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.27 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.28 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.29 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.3 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.30 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.31 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.32 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.33 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.34 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.35 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.4 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.5 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.6 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.7 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.8 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - contao/core 3.5.9 conflicts with contao/managed-edition[No version set (parsed as 1.0.0)].
    - Installation request for contao/managed-edition No version set (parsed as 1.0.0) -> satisfiable by contao/managed-edition[No version set (parsed as 1.0.0)].
    - Installation request for exploreimpact/contao-i18nl10n ^1.5 -> satisfiable by exploreimpact/contao-i18nl10n[1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.5.4, 1.5.5].

<warning>Running update with --no-dev does not mean require-dev is ignored, it just means the packages will not be installed. If dev requirements are blocking the update you have to resolve those problems.</warning>
Finished Composer Cloud resolving.

```

**Comment by ClaudioDeFacci** [comment]
Did you install the package with the version specified?
`composer require exploreimpact/contao-i18nl10n:2.0.0-beta`

**Comment by zonky2** [comment]
I try it per contao manager
it´s not possible to choose a version

**Comment by ClaudioDeFacci** [comment]
Ah, I thought it is possible to choose a version: https://docs.contao.org/books/manager/de/erweiterungen-verwalten.html#pakete-installieren
I am used to work with composer directly and almost never used the Contao Manager.
I will check that as soon as possible.

**Comment by zonky2** [comment]
Since I know this from many programmers, I check if this via contao manager with ;-)

**Comment by zonky2** [comment]
why closed? fixed?

**Comment by exploreimpact** [comment]
It is most likely fixed as versions 2.0.x exist.
But i generally just removed all issues to get rid of all old topics. I moved the most important issues to a GitHub project board.
